app/src/main/cpp/skyline/vfs/ticket.cpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#include "ticket.h"
+
+namespace skyline::vfs {
+    enum class SignatureType : u32 {
+        Rsa4096Sha1 = 0x010000,
+        Rsa2048Sha1 = 0x010001,
+        EcdsaSha1 = 0x010002,
+        Rsa4096Sha256 = 0x010003,
+        Rsa2048Sha256 = 0x010004,
+        EcdsaSha256 = 0x010005
+    };
```

Comment:
We tend to keep structures and enumerations in the header file, there's few exceptions to this with structures but enumerations should always be in the header with an appropriate comment, which would be the following in this case:
```cpp
/**
  @url https://switchbrew.org/wiki/Ticket#Signature_type
 **/
```
It should also be noted that we tend to have a trailing comma after our enumerations

Reply:
I disagree with this, enumerations in c++ makes more sense to me